### DISCUSSION TITLE
How to install AGE in the Windows edition of the PostgreSQL?

### DISCUSSION THREAD

**Discussion by fdsmax** [apache/age#583] [Q&A]
Hi,

How to install AGE in the Windows edition of the PostgreSQL? I could not find any notes or reference on the Windows support for AGE. Any pointers will be helpful.

Thanks and regards

**Comment by titoausten** [comment]
AGE isn't compatible with Windows but you can bypass that by using WSL. Kindly click [here](https://dev.to/titoausten/easy-guide-to-install-postgresql-from-source-code-on-windows-2k60) for more details and clarity on that. If you have more questions, I'm available to answer you.

**Comment by markgomer** [comment]
This site has a Windows version installer available now:

https://agedb.io/community.jsp

**Comment by AffanBitnine** [comment]
This site has a Windows version installer available now: Version age-plus-pg15 windows x64
https://drive.google.com/file/d/1SZ-IpgXChATOtG8U1Agl3CVqllJE4s8d/view?usp=drive_link
